cosmos-gui/views/dashboard.jade

```diff
@@ -1,6 +1,31 @@
+//
```

Comment:
Once again, is using Jade really needed? I guess it is because you are leaning on something based on Jade... :) I am just saying this because it makes people needing to know Jade to contribute, debugging is more complicated, etc... :) I am not saying that Jade should not be used, just it should be really justified... :)